pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/PulsarClientTool.java

```diff
@@ -76,6 +76,9 @@ public static class RootParams {
 
         @Parameter(names = { "--tlsTrustCertsFilePath" }, description = "File path to client trust certificates")
         String tlsTrustCertsFilePath;
+
+        @Parameter(names = { "-m", "--memory", }, description = "Configure the Pulsar client memory limit")
+        Long memory=0L;
```

Comment:
It's pretty unconvenient to specify megabytes as bytes on the command line. Do we already have a parser that can accept typical units like M = megabyte etc. .

Reply:
Do you think it should be changed to this ？

@Parameter(names = { "-M", "--Memory", },

Reply:
@shy-share IIUC it means you can specify `-m 41M` means configuring memory as 41MB.

Reply:
@tisonkun I know what you mean but I don't know how to write this

Reply:
@lhotari @nicoloboschi Does our CLI tools already provide such functionality? I can read `broker.conf` options like:

```ini
# For Amazon S3 ledger offload, Max block size in bytes. (64MB by default, 5MB minimum)
s3ManagedLedgerOffloadMaxBlockSizeInBytes=67108864

# For Amazon S3 ledger offload, Read buffer size in bytes (1MB by default)
s3ManagedLedgerOffloadReadBufferSizeInBytes=1048576
```